report_thesis/src/sections/methodology.tex

```diff
@@ -97,31 +97,58 @@ \subsection{Data Preparation}
-Realizing the potential of this, we decided to further investigate how the predictive accuracy and robustness could be improved by combining the strengths of these models and preprocessing techniques.
-From our investigation, we determined that the stacking ensemble method seemed very promising and novel in the LIBS analysis field, leading us to select it as the primary method for our study.
-
-Stacking ensemble is analogous to the methodologies employed in the original \gls{moc} pipeline, which also tailored predictions for each oxide by blending outputs from the \gls{pls1-sm} and \gls{ica} phases, variably weighting the influence of the \gls{ica} predictions depending on the oxide.
-Unlike the \gls{moc} model, which required manual determination of the model weightings for each oxide, our method utilizes a meta learner to learn optimal parameter settings.
-Stacking ensemble is beneficial as it dynamically adapts to our dataset's characteristics without the need for domain-specific knowledge.
-This approach represents a more sophisticated method, streamlining complex model configurations and potentially enhancing predictive accuracy through dynamically learned integrations, rather than fixed presets.
-
+Choosing the right models and preprocessing techniques for \gls{libs} data analysis is a challenging task. 
+The literature suggests a variety of models and preprocessing techniques that promise to be adept at handling data that exhibits high-dimensionality, multi-collinearity, and matrix effects.
+Additionally, different machine learning models perform best on different oxides in LIBS data due to the complex nature of the data. 
+These challenges and model-specific strengths suggests that an optimal approach would involve combining multiple models. 
+This notion is supported by the advent of models such as the \gls{moc}~\cite{cleggRecalibrationMarsScience2017}, which combines the predictions of multiple models using a predefined weighting of the predictions made of each model, per-oxide.
+This approach improved the accuracy over the individual models, but required manual tuning of the weights for each model.
+The manual tuning poses limitations, both in terms of the analysis required to determine a weighting, as well as, the risk for suboptimal weighting.
+Based on these considerations, it seems reasonable to consider techniques that could automate such a process, while still being able to leverage the strengths of multiple models.
+To address these criteria, we chose to adopt a stacking ensemble approach. 
+As described in Section~\ref{subsec:stacked-generalization}, is a method that utilizes multiple base estimators trained on the same data, whose predictions are then used to train a meta-learner.
+This approach has the benefit of being able to leverage the strengths of multiple models on a per-oxide basis in conjunction with the most beneficial preprocessors for each of these models.
```

Comment:
Well, that is really a general ensemble benefit for multiple, different regression targets. Not so much a stacking benefit. The benefit of stacking is that it could automate and potentially improve upon the manual tuning by using a meta-learner to discern patterns in the base predictors' outputs.